definitions/npm/ioredis_v3.x.x/flow_v0.46.x-/ioredis_v3.x.x.js

```diff
@@ -12,6 +12,9 @@
     const redis = new Redis();
  =============================================== */
 
+import {callback} from "../../awaiting_v2.x.x/flow_v0.33.x-/awaiting_v2.x.x";
```

Comment:
You can't import other type definitions, see the contributing guide.

Reply:
fixed :-)